packages/core/src/Operation.ts

```diff
@@ -137,6 +137,8 @@ export const toLambda = (operation: Operation): MichelsonV1Expression[] => {
             operation.recipient.pkh,
             Number(operation.amount)
           );
+        default:
+          throw new Error(`${operation.recipient.type} is not supported yet`);
```

Comment:
is it though?

Reply:
sending TEZ to rollup isn't supported at the moment